galaxy_ng/tests/integration/utils.py

```diff
@@ -375,7 +375,20 @@ def set_certification(client, collection):
             f"{collection.version}/move/staging/published/"
         )
 
-        client(url, method="POST", args=b"{}")
+        job_tasks = client(url, method="POST", args=b"{}")
+        assert 'copy_task_id' in job_tasks
+        assert 'curate_all_synclist_repository_task_id' in job_tasks
+        assert 'remove_task_id' in job_tasks
+
+        task_ids = list(job_tasks.values())
+        task_ids = [x for x in task_ids if x]
```

Comment:
will this be of arbitrary order? not sure if we care to try to wait on copy first... i guess if we wait on remove first and copy fails then remove task will eventually timeout i assume

Reply:
I don't know that the order really matters. Does an end user or client know which order those tasks should start and complete in?

Reply:
no it doesn't. but for testing, if copy fails, remove will likely not be `complete` it might be `canceled` ... nice for a test to fail as close as possible to the failing code.